CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## master (unreleased)
 
+### New features
+
+* [#5319](https://github.com/bbatsov/rubocop/pull/5319): Add new `Security/Open` cop. ([@mame][])
```

Comment:
Can you add `[@mame]: https://github.com/mame` to bottom of the file?

Reply:
`git push --force` done!